src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    //TODO make methods for the different operations
+    protected static Task[] list = new Task[100];
+    public static final String OPENING_MSG = "Hello! I'm Dukebot\n\tWhat can I do for you?";
+    public static final String CLOSING_MSG = "Goodbye! Hope to see you again soon ^^!";
+    public static final String HORIZONTAL = "---------------------------------";
+    public static final String MARK_MSG = "Nice! I've marked this task as done:";
+    public static final String UNMARK_MSG = "Oki! I've marked this task as not done yet:";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(HORIZONTAL + "\n\t" + OPENING_MSG + "\n" + HORIZONTAL);
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine().toLowerCase();
+
+        int listNum;
+
+        int item = 0;
+
+        while (!line.equals("bye")) {
+            if (!(line.equals("list") || line.startsWith("mark") || line.startsWith("unmark"))) {
```

Comment:
Perhaps this if statement can be refactored out to the bottom as a "default" case. Further, the condition expression is quite long! 

In the code below, checks have already been accounted for:
`list, mark, unmark`.

Since all 3 commands have been checked already, the condition evaluated in this line:  
`if (!(line.equals("list") || line.startsWith("mark") || line.startsWith("unmark"))`  

Is implicitly satisfied from line 58 onwards. So, you can consider factoring out the logic for the addition of a new Task to line 58!

Reply:
Thanks will consider refactoring that in my code!